source/_posts/2017-05-19-kanban-method-in-a-consulting-company.md

```diff
@@ -0,0 +1,128 @@
+boards in [Trello](https://trello.com/), which streamlines our work in kanban.
+Our boards are divided into appropriate categories: Backlog, To Do, In Progress,
+Verification, Done and Project Status, where we can discuss project features
+or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According to the Kanban method we give out a kanban cards. Every card must have:
+
+* Title - As needed, it can be very general or low-level,
+* Description - With some documentation, links or to do list,
+* Assigne - If you want to plan something with others You can assing more than
+one person, but we recommend to assing one person / per task,
+* Label - According to the task,
+* Due date - We set a maximum of two weeks to complete the tasks.
+
+The card is very important in the whole project process. Thanks to
+her, we measure our work and set time for each stage in the design, develop
+and testing solutions in embedded systems projects. Additionally inside the
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+
+The card life begins in Backlog, tasks here's have lowest priority and they
+waiting for select to development - but it's not a rule, sometimes we add cards
+with high priority directly to ToDo column. ToDo is the queue for the most
+important column - In Progress. From there our goal is to safely move this task
+to done. Of course sometimes something will happen, but by analyzing the boards
+we are able to find the blocker very quickly. In embedded systems company, we
+are dealing mostly with blockers in the form not working or missing hardware,
+incompatibility of the kernel with hardware, frequent changes in priorities.
+In case of problems we try to react quickly to not disturb the whole process.
+
+
+![kanban-flow](http://imageshack.com/a/img923/2471/u20Zzr.jpg)
+
+## Rules
+
+To make kanban work, each employee must follow these rules:
+
+* In Progress have biggest priority, next is ToDo and Backlog,
+* Never take more than two tasks at a time - it is interesting to note that when
+switching between two tasks we need about 13 minutes to return to the previous
+concentration,
+* Do not throw unfinished tasks for verification, because due date is comming,
+* Listen and read communication, ask questions - takes priority tasks no focus
+on backlog,
+* Discuss inside cards - about problems and progress,
+* Suggest changes in places where you see such a need.
+
+These rules are only a sample but improve the whole process. We know situations
+where tasks is going fast to verification, after which it turns out that the
+task is incomplete and we need turn it back to the left. When the checks show
+such errors, we try to first determine the degree of completion of the card, and
+then decide whether the card must return or it is a matter of a small patch.

```

Comment:
@molszewski9412 sometimes we split cards and move part of result to Verification and other part to ToDo.